src/app/auth/features/user-manager-detail/user-manager-detail.component.html

```diff
@@ -0,0 +1,96 @@
+@if (user(); as currentUser) {
+  <div class="container mx-auto p-6">
+    <div class="clearfix mb-6">
+      <button
+        mat-icon-button
+        (click)="goBack()"
+        aria-label="Go back to user list"
+        class="text-primary float-left mt-1 mr-4"
+      >
+        <mat-icon>chevron_left</mat-icon>
+      </button>
+      <h1 class="text-3xl font-bold">User: {{ currentUser.displayName }}</h1>
```

Comment:
```suggestion
      <h1 class="font-bold">User: {{ currentUser.displayName }}</h1>
```
I think this is redundant